### PR TITLE
Add TED to Bid Tracker/List for a Bid

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**

### CODE REVIEW

**Comment by mjoyce91** [comment]
@elizabeth-jimenez to get tests to pass, you can run `yarn test --updateSnapshot --cache --runInBand=false`. The `--updateSnapshot` option forces the snapshots to update.

File: src/Components/BidTracker/BidTrackerCardTitle/BidTrackerCardTitle.jsx [hunk 1]
```diff
@@ -58,6 +59,14 @@ const BidTrackerCardTitle = ({
               <BidCount bidStatistics={bidStatistics} altStyle />
             </span>
         }
+        {
+          !condensedView &&
+          <div className="bid-tracker-card-title-bottom">
```
Comment (mjoyce91) [inline src/Components/BidTracker/BidTrackerCardTitle/BidTrackerCardTitle.jsx]: What if we use the Static Content wrapper here? Then we can hide this with the feature flag.

```jsx
import StaticDevContent from 'Components/StaticDevContent';
...
<StaticDevContent>
  <div className="bid-tracker-card-title-bottom">
</StaticDevContent>
...
```

**Comment by elizabeth-jimenez** [comment]
> @elizabeth-jimenez to get tests to pass, you can run `yarn test --updateSnapshot --cache --runInBand=false`. The `--updateSnapshot` option forces the snapshots to update.

running now!

File: src/Components/BidTracker/BidTrackerCardTitle/BidTrackerCardTitle.jsx [hunk 2]
```diff
@@ -58,6 +59,14 @@ const BidTrackerCardTitle = ({
...
+            <strong>TED:</strong> {formatDate('2020-07-02T05:00:00Z')}
```
Comment (mjoyce91) [inline src/Components/BidTracker/BidTrackerCardTitle/BidTrackerCardTitle.jsx]: Good catch to use our `formatDate` utility 👍

File: src/Components/BidTracker/BidTrackerCardTitle/BidTrackerCardTitle.jsx (hunk 1 above)
Comment (elizabeth-jimenez) [reply src/Components/BidTracker/BidTrackerCardTitle/BidTrackerCardTitle.jsx]: will do!

Comment (mjoyce91) [inline src/Components/BidTracker/BidTrackerCardTitle/BidTrackerCardTitle.jsx]: Might not have been clear in the ticket, but can we move it to this section?

Screen Shot 2020-01-09 at 4 17 33 PM

Comment (elizabeth-jimenez) [reply src/Components/BidTracker/BidTrackerCardTitle/BidTrackerCardTitle.jsx]: No problem. I was wondering if we had a mock-up somewhere that we are following?

**Comment by rtirserio** [comment]
@elizabeth-jimenez click on the details link next to the failing check so you can see the issue and fix it

**Comment by elizabeth-jimenez** [comment]
@rtirserio last commit passed all checks

**Pull request merged**